pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tensorlake"
-version = "0.2.17"
+version = "0.2.18"
```

```suggestion
version = "0.2.19"
```

I've merged a PR with the 0.2.18 version before this 😄